components/eam/src/physics/rrtmgp/radiation.F90

```diff
@@ -1392,9 +1392,9 @@ subroutine radiation_tend(state,   ptend,    pbuf,          cam_out, cam_in,  &
                call handle_error(clip_values(cld_tau_gpt_sw,  0._r8, huge(cld_tau_gpt_sw), trim(subname) // ' cld_tau_gpt_sw', tolerance=1e-10_r8))
                call handle_error(clip_values(cld_ssa_gpt_sw,  0._r8,                1._r8, trim(subname) // ' cld_ssa_gpt_sw', tolerance=1e-10_r8))
                call handle_error(clip_values(cld_asm_gpt_sw, -1._r8,                1._r8, trim(subname) // ' cld_asm_gpt_sw', tolerance=1e-10_r8))
```

Comment:
dumb question, but shouldn't we (1:ncol,:,:) all of these handle_error calls rather than just the aer_ ones? Or are the cld ones not 3d variables?

Reply:
I only know that these were the line numbers of crashes -- ie, I changed one line, ran again, and got error at another location with `aer_*` variable clipping.

Reply:
Yeah, you’ll likely run into the same problem here.